src/PieChart.js

```diff
@@ -95,6 +95,12 @@ class PieChart extends React.Component {
      * or accessor function which returns a class.
      */
     pieSliceClassName: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
+    /**
+     * Inline style object applied to each pie slice.
+     * When a function is provided it will receive the value for the slice and should return the
```

Comment:
I believe we're also giving users the index as well so maybe we can write
"When a function is provided it will receive the value and index for the slice as its parameters and should return a style object that will be applied to that slice."